modules/ROOT/pages/_partials/app-security/setting-up-the-app-security-sdk.adoc

```diff
@@ -0,0 +1,43 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+include::configuring-dev-env-app-security.adoc[]
+
+== Setting up {app-security-service} service SDK
+
+=== Importing the libraries
+
+[tabs]
+====
+
+// end::excludeDownstream[]
+Cordova::
++
+--
+
+. Install the link:https://www.npmjs.com/package/@aerogear/security[AeroGear Security] package from link:https://www.npmjs.com/[NPM^]
++
+[source,bash]
+----
+$ npm install @aerogear/security
+----
+
+. Add the link:https://www.npmjs.com/package/@aerogear/cordova-plugin-aerogear-security[AeroGear Security plugin^] for Cordova:
++
+[source,bash]
+----
+$ cordova plugin add @aerogear/cordova-plugin-aerogear-security
+----
+
+. Import and instantiate `SecurityService` to start using the SDK:
++
+[source,javascript]
+----
+const SecurityService = require("@aerogear/security");
```

Comment:
This is an older way of importing I think, and in the showcase app you import like this:

```ts
import { AppSecurity } from '@aerogear/security';
```

Reply:
Matching the existing doc's